src/popup/router/components/SidebarMenu.vue

```diff
@@ -10,7 +10,7 @@
         <div class="f-12" v-if="account.name" data-cy="chain-name">
           <TruncateMid :str="account.name" />
         </div>
-        <div class="account-type">{{ $t('mainAccount') }}</div>
+        <div class="account-type">{{ account.userName }}</div>
```

Comment:
`userName` is confusing, it sounds like the account is referencing some user by name. Also, aens names are also defined by the user. Proposing to use `localName` instead.